client/deis.py

```diff
@@ -1321,24 +1322,34 @@ def config_push(self, args):
         """
         Sets environment variables for an application.
 
-        Your environment is read from a file named .env. This file can be
-        read by foreman to load the local environment for your app.
+        Your environment is read from the optionally specified path to a
```

Comment:
Let's change this to

"The environment is read from `<path>`. This file can be read by foreman to load the local environment for your app."